stdlib/LinearAlgebra/src/bidiag.jl

```diff
@@ -254,6 +254,7 @@ end
 adjoint(B::Bidiagonal) = Adjoint(B)
 transpose(B::Bidiagonal) = Transpose(B)
 adjoint(B::Bidiagonal{<:Real}) = Bidiagonal(B.dv, B.ev, B.uplo == 'U' ? :L : :U)
+adjoint(B::Bidiagonal{<:Number}) = Bidiagonal(conj.(B.dv), conj.(B.ev), B.uplo == 'U' ? :L : :U)
```

Comment:
```suggestion
adjoint(B::Bidiagonal{<:Number}) = Bidiagonal(conj(B.dv), conj.(B.ev), B.uplo == 'U' ? :L : :U)
```
If you use `conj` without the dot, then it doesn't make a copy for the `Real` case.